wp-content/themes/workday/less/vars.less

```diff
@@ -2,6 +2,7 @@
 // if updating these colors, please update inc/block-color-palette.php
 @green: #71E26C; // for use on dark backgrounds;
 @darkgreen: #16A327; // for use on light backgrounds;
+@mint-green: #7BE2B6;
```

Comment:
Should we use plain `@green` here?

Reply:
In the design on Xd it's `#7BE2B6` so I'm not sure. @MirandaEcho?

For reference: https://xd.adobe.com/spec/e12ba5e6-9fe1-448a-68e4-e701056351e0-971a/screen/b31b36e7-9b99-4a04-b37a-572cb5815bf0/Article-Revised

Reply:
Those mockups have like eight different shades of green.

Reply:
Whatever is closest/close-ish to the mockups is fine.